controller/share_manager_controller.go

```diff
@@ -1050,6 +1045,92 @@ func (c *ShareManagerController) getShareManagerTolerationsFromStorageClass(sc *
 	return tolerations
 }
 
+func (c *ShareManagerController) validateStorageNetworkForRWXVolume() (bool, error) {
```

Comment:
Should using a similar naming convention to `checkStorageNetworkApplied`, for example:
```suggestion
func (c *ShareManagerController) checkStorageNetworkForRWXVolumeIsValid() (bool, error) {
```